homeassistant/components/everlights/light.py

```diff
@@ -79,63 +79,35 @@ class EverLightsLight(LightEntity):
     _attr_supported_color_modes = {ColorMode.HS}
     _attr_supported_features = LightEntityFeature.EFFECT
 
-    def __init__(self, api, channel, status, effects):
+    def __init__(
+        self,
+        api: pyeverlights.EverLights,
+        channel: int,
+        status: dict[str, Any],
+        effects,
```

Comment:
@flexxor do you know what the type hint should be for `effects`?
I couldn't work out what `await api.get_all_patterns()` returns...
Is it a list? is it a dict? can it be None?...

Reply:
Unfortunately not, sorry.

Reply:
Thanks, I think I'll leave it undefined until someone figures it out.